src/modules/registrypreview/RegistryPreviewUI/RegistryPreviewXAML/MainWindow.xaml

```diff
@@ -95,7 +95,7 @@
                     x:Uid="OpenButton"
                     Click="OpenButton_Click">
                         <AppBarButton.Icon>
-                            <FontIcon Glyph="&#xe8e5;"/>
+                            <FontIcon FontFamily="{StaticResource SymbolThemeFontFamily}" Glyph="&#xe8e5;"/>
```

Comment:
Can't hurt to declare it, but it's default and thus redundant.
See https://learn.microsoft.com/nl-nl/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.fonticon?view=windows-app-sdk-1.2#default-font-family

Reply:
I keep going back to this from when I made the change, regardless of what the docs tell us:
> "I agree it should, but the fallback behavior with zh-Hans hasn't, well, fallen back as it should. On Win11, it should use Segoe Fluent Icons, but on Win10 it should look for Segou MDL2 Assets, and if Windows 8 was supported, it should look for Segoe UI Symbol. For LIXiangChen, Windows 10 with Simplified Chinese, the original code picked up an unexpected font for the icon, which is the oddity. My hope is that by calling out the StaticResource explicitly, it will behave better."\

I'll think it's better to have it.